src/test/common/clara/test_testing_utils.cljc

```diff
@@ -0,0 +1,56 @@
+#?(:clj
```

Comment:
Yeah, doing it all in one body with reader conditionals is also an option.  In this case I just thought that so much differed for such a small ns header that it would be more readable to just have entirely separate forms.  Looking back at it I'm thinking perhaps make the namespace name common and just have the rest be separate.